kclvm/sema/src/advanced_resolver/node.rs

```diff
@@ -867,6 +867,42 @@ impl<'ctx> MutSelfTypedResultWalker<'ctx> for AdvancedResolver<'ctx> {
     }
 
     fn walk_config_expr(&mut self, config_expr: &'ctx ast::ConfigExpr) -> Self::Result {
+        let (start, end) = (self.ctx.start_pos.clone(), self.ctx.end_pos.clone());
+        let schema_symbol = self.ctx.schema_symbol_stack.last().unwrap_or(&None).clone();
+        let kind = LocalSymbolScopeKind::Config;
+
+        self.enter_local_scope(
+            &self.ctx.current_filename.as_ref().unwrap().clone(),
+            start.clone(),
+            end.clone(),
+            kind,
+        );
+
+        let cur_scope = *self.ctx.scopes.last().unwrap();
+        self.gs
+            .get_scopes_mut()
+            .set_owner_to_scope(cur_scope, schema_symbol.unwrap());
+
+        if let Some(schema_expr) = self
+            .gs
+            .get_symbols_mut()
+            .exprs
+            .get_mut(schema_symbol.unwrap().get_id())
+        {
+            let mut expr_symbol = ExpressionSymbol::new(
+                schema_expr.name.to_string(),
+                start.clone(),
+                end.clone(),
+                None,
+            );
+            expr_symbol.hint = Some(SymbolHint {
+                pos: start.clone(),
+                kind: SymbolHintKind::VarHint(schema_expr.name.to_string()),
+            });
+        }
```

Comment:
you only created a new symbol, but you didn’t add it to the context. You need `self.gs.get_symbols_mut().alloc_expression_symbol()`